test/unit/coffee/HttpControllerTests.coffee

```diff
@@ -292,6 +293,20 @@ describe "HttpController", ->
 					.calledWith(400)
 					.should.equal true
 
+		describe "when the doc body is too large", ->
+			beforeEach ->
+				@req.body =
+					lines: @lines = Array(2049).fill('a'.repeat(1024))
+					version: @version = 42
+					ranges: @ranges = { changes: "mock" }
+				@HttpController.updateDoc @req, @res, @next
+
+			it "should return a 413 (too large) response", ->
+				sinon.assert.calledWith(@res.status, 413)
+
+			it "should report that there are too many lines", ->
```

Comment:
minor fix to test description: "should report that the document body is too large"

Reply:
:+1: thanks. Will change this and squash.